submissions/best-of-its-genre-nanogamer7/origins.md

```diff
@@ -0,0 +1,46 @@
+
+So I went on a search (which definitely wasn't easy, seeing how active some of Wube's devs are across Reddit, Discord and the forum), and after sifting through a few interviews and clicking on google page 2, I finally found something: An [old forum thread](https://forums.factorio.com/viewtopic.php?f=5&t=3026) from March 2014, confirming Buildcraft (or IndustrialCraft, according to a different interview) as the main inspiration to Factorio.[^5][^6] According to kovarex, he saw the potential in that concept while playing the mod, and wanted more of it (in his own spin, of course).
+
+While I had hoped to find an answer, I for sure wasn't expecting to find such a direct one. Having that out of the way, though, we can definitely speculate a bit more on brought Factorio to were it's at today.
+
+### Great games design themselves
+
+There is a principle in game design designers should follow, that goes along the lines "follow the fun",[^7] meaning you shouldn't try to force a game to be something it isn't, but rather develop on the ideas that are enjoyable. While examples for that statement often include dramatic changes like Ape Out went through,[^8] it still applies to games that don't shift that drastically in direction. Let's take Factorio itself as an example:  Originally the game had multiple pickaxes – a wooden, which you could then craft into a better one out of stone, and then into the iron pickaxe we had for such a long time [^citation needed]. This was clearly inspired by Minecraft (and also confirmed by the devs[^9]), but having so many items about manual mining in a game about automation gives the idea about the game – it draws the fun away. So they simplified it, gave you just your iron and steel pickaxe, but that's still two items specifically about manual mining. Thus they removed pickaxes completely from the game in `0.17.0`[^10], and with all focus on the unfun, manual labor.
+
+You might ask, why did I dive into a concept of game design now? Well I believe it is an important step on understanding how Factorio came to be. I believe, once they core gameplay concept of a game is clear – which is what Factorio started out with – it's just a manner of fitting features to it. Automation? That sounds like the industrial revolution, when they invented belts. Need something to get that thing of the belts? How about inserters, they are at least not human. You get the point. A game doesn't need inspiration from other games to develop its ideas.
+
+### Conclusio
+
+While I originally tried to find Factorios "predecessor", it stemming itself with only  the core gameplay loop as an external influence makes this conclusion even more rewarding. That being said, there have probably been more automation games in the past, and Factorio definitely had a big influence on some more recent games. Maybe we can explore this further in a future article, maybe even how Factorio inspired more than just games. You'll see ;)
```

Comment:
> While I originally tried to find Factorios "predecessor", it stemming itself with only the core gameplay loop as an external influence makes this conclusion even more rewarding

I'm not sure what you're trying to say here?

Reply:
"predecessor" as in Infiniminer for Minecraft, Warcraft for Starcraft, etc. but Factorio doesn't have that, it only has the idea of "automation", and it not having a predecessor as descibed above feel like a more satisfying conclusion to me than just finding that predcessor.
I'm explaining this here as I have no idea how to bring this nuance between "origin" and "predecessor" in a different way to text than I already did, and was hoping you might have a better idea 😬